lib/src/module.dart

```diff
@@ -139,6 +139,8 @@ abstract class Module {
       String? definitionName, bool reserveDefinitionName) {
```

Comment:
Probably want to rename these to `name` and `reserveName` since they could be used for `definitionName` or `name`?